src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockGoogleServicesDAO.scala

```diff
@@ -94,12 +95,18 @@ class MockGoogleServicesDAO extends GoogleServicesDAO {
   override def getDownload(bucketName: String, objectKey: String, userAuthToken: WithAccessToken)
                           (implicit executionContext: ExecutionContext): Future[PerRequestMessage] = {Future.successful(RequestComplete(StatusCodes.NotImplemented))}
   override def fetchPriceList(implicit executionContext: ExecutionContext): Future[GooglePriceList] = {
-    Future.successful(GooglePriceList(GooglePrices(UsPriceItem(BigDecimal(0.01)), UsTieredPriceItem(Map(1024L -> BigDecimal(0.12)))), "v0", "18-November-2016"))
+    Future.successful(GooglePriceList(GooglePrices(Map("us" -> 0.01, "europe-west1" -> 0.02), UsTieredPriceItem(Map(1024L -> BigDecimal(0.12)))), "v0", "18-November-2016"))
```

Comment:
Nit: multi-regional is generally expected to be more expensive than regional. Looking at https://cloud.google.com/storage/pricing#price-tables , for standard storage US is 0.026/GB, and europe-west1 is 0.02. 

I do understand that this is just a mock test though, so not too big a deal.